cognite/client/_api/data_modeling.py

```diff
@@ -13,4 +16,7 @@
 class DataModelingAPI(APIClient):
     def __init__(self, config: ClientConfig, api_version: Optional[str], cognite_client: CogniteClient) -> None:
         super().__init__(config, api_version, cognite_client)
+        self.containers = ContainersAPI(config, api_version, cognite_client)
+        self.data_models = DataModelsAPI(config, api_version, cognite_client)
```

Comment:
Just reiterating here from slack: I would prefer this to be just `models` so that the full call from the user won't have `data_` repeated.

Reply:
Agreed. I will change it.